blueprints/pix-component/files/addon/stories/pix-__name__.js

```diff
@@ -0,0 +1,23 @@
+import { hbs } from 'ember-cli-htmlbars';
+
+export const stars = (args) => {
```

Comment:
Il faut mettre le nom du composant ici:
```suggestion
export const <%= classifiedModuleName %> = (args) => {
```